src/client/Microsoft.Identity.Client/Internal/Broker/BrokerInteractiveRequest.cs

```diff
@@ -10,47 +10,85 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.Identity.Client.Internal.Broker
 {
-    internal class BrokerInteractiveRequest
+    internal class BrokerInteractiveRequest : InteractiveRequest
```

Comment:
TLDR; it should be cleaner to derive from `RequestBase`

What does RequestBase provide? 
- access to ServiceBundle, config params and request params
- scope validation
- logging and telememetry

and hooks for: 

- caching token response -> possibly used by broker request
- resolve authority 
- send request to /token endpoint with headers, body params -> not used by broker